code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -22,12 +15,13 @@
 	max_integrity = 100
 
 	var/stat = CONSCIOUS //UNCONSCIOUS is the idle state in this case
-
 	var/sterile = FALSE
 	var/real = TRUE //0 for the toy, 1 for real. Sure I could istype, but fuck that.
 	var/strength = 5
-
 	var/attached = 0
+	var/impregnation_time = 12 SECONDS //Time it takes for facehugger to deposit its eggs and die.
+	var/min_active_time = 200 //time between being dropped and going idle
+	var/max_active_time = 400
```

Comment:
Any reason not to make these seconds as well? Also, maybe autodoc these for good measure